css/style.css

```diff
@@ -137,7 +140,7 @@ h4 {
   margin-top: 5%;
 }
 .social_icons {
-  margin-left: 30px;
+  margin-left: 20px;
```

Comment:
Consider using relative units (em, vw, vh etc) to improve scaling in different screen sizes?

Reply:
check if `margin-left: 20px;` used in CSS is necessary as it might affect element horizontal layout
? Use horizontal / vertical alignments (Flexbox / text-alignment / vertical-alignment) to adjust icons if necessary.